QGL/drivers/APS2Pattern.py

```diff
@@ -1390,7 +1390,8 @@ def tdm_instructions(seqs):
 
             elif isinstance(s, PulseSequencer.Pulse):
                 if s.label == 'MEAS' and s.maddr != (-1, 0):
-                    instructions.append(CrossBar(2**s.maddr[1], 0x1, label=label))
+                    tdm_chan = m.channel.processor_chan.channel if getattr(m.channel, 'processor_chan') else 1
+                    instructions.append(CrossBar(2**m.maddr[1], 2**(tdm_chan-1)))
```

Comment:
Why do we not need to pass along the label here?

Reply:
I think we do, I'll add it back